src/vsphere_cpi/lib/cloud/vsphere/stemcell.rb

```diff
@@ -80,6 +80,13 @@ def replicate(datacenter, cluster, to_datastore, datastore_cluster = nil)
         replica_vm = client.wait_for_task do
           vm.clone(datacenter.template_folder.mob, replica_name, clone_spec)
         end
+        disable_sdrs(srm, datastore_cluster.mob, replica_vm) if datastore_cluster
```

Comment:
Do you think adding a log message here about disabling sdrs for a stemcell will be useful to admins who are debugging?

Reply:
Its already present inside disable_sdrs method